shell/src/main/java/org/apache/accumulo/shell/commands/CloneTableCommand.java

```diff
@@ -99,6 +105,8 @@ public Options getOptions() {
     noFlushOption =
         new Option("nf", "noFlush", false, "do not flush table data in memory before cloning.");
     o.addOption(noFlushOption);
+    keepOfflineOption =
+        new Option("k", "keepOffline", false, "do not bring the table online after cloning.");
```

Comment:
Probably better to name the option "o", to be consistent with other offline commands.  For example, that is what [CreateTableCommand](https://github.com/apache/accumulo/blob/3fd5cad92f9b63ac19e4466f3f2d5237b905262c/shell/src/main/java/org/apache/accumulo/shell/commands/CreateTableCommand.java#L328) uses in 2.X

Reply:
done